src/WP_Export_Query.php

```diff
@@ -368,10 +371,26 @@ private function check_for_orphaned_terms( $terms ) {
 
 		foreach ( $have_parent as $has_parent ) {
 			if ( ! isset( $term_ids[ $has_parent->parent ] ) ) {
-				$this->missing_parents = $has_parent;
-				throw new WP_Export_Term_Exception( "Term is missing a parent: {$has_parent->slug} ({$has_parent->term_taxonomy_id})" );
+				if ( $this->filters['allow_orphan_terms'] ) {
+					$orphans[ $has_parent->term_id ] = true;
+				} else {
+					$this->missing_parents = $has_parent;
+					throw new WP_Export_Term_Exception( "Term is missing a parent: {$has_parent->slug} ({$has_parent->term_taxonomy_id})" );
+				}
+			}
+		}
+
+		if ( count( $orphans ) > 0 ) {
```

Comment:
For the sake of clarity and avoiding accidental breakage, can we return `$terms` early if `! $this->filters['allow_orphan_terms']`?

Reply:
@danielbachhuber Updated here: https://github.com/wp-cli/export-command/pull/109/commits/218ac7528957413b0d99b57f0de8072d27b41574